paasta_tools/cli/cmds/logs.py

```diff
@@ -329,6 +329,9 @@ def paasta_app_output_passes_filter(
     # https://github.com/gweis/isodate/issues/53
     except ValueError:
         return True
+    except AttributeError:
+        # timestamp might be missing at all
```

Comment:
can we add a little more context about why they might be missing just to prevent someone from looking at logs in the future, seeing that they all have a timestamp field, and then deleting this block!

e.g., `vector/otel-collector will split long log-lines and we might end up with entries that have missing fields - see TICKET-XXXX`